evergreen/evergreen.yml

```diff
@@ -202,12 +202,32 @@ functions:
       params:
         script: |
           ${PREPARE_SHELL}
-          MONGODB_VERSION=${VERSION} TOPOLOGY=${TOPOLOGY} AUTH=${AUTH} SSL=${SSL} sh ${DRIVERS_TOOLS}/.evergreen/run-orchestration.sh
+          MONGODB_VERSION=${VERSION} \
+          TOPOLOGY=${TOPOLOGY} \
+          AUTH=${AUTH} \
+          SSL=${SSL} \
+          ORCHESTRATION_FILE=${ORCHESTRATION_FILE} \
+            sh ${DRIVERS_TOOLS}/.evergreen/run-orchestration.sh
     # run-orchestration generates expansion file with the MONGODB_URI for the cluster
     - command: expansions.update
       params:
         file: mo-expansion.yml
 
+  ocsp-bootstrap-mongo-orchestration:
```

Comment:
can you please describe what happens in the below(all :) ) configuration sections

Reply:
I think for the EG file, it'd be quicker to have a walkthrough over Zoom on Monday! Perhaps after triage?

Reply:
Discussed offline!